docs/INSTALLATION.md

```diff
@@ -1,13 +1,27 @@
 # Installation
 
-Install the essential build tools, the QEMU emulator, and the GCC ARM64 cross-toolchain to test your code.
+## Ubuntu
+
+The essential build tools can be installed using
 
 ```shell
-sudo apt install build-essential qemu-user qemu-user-static binutils-aarch64-linux-gnu gcc-aarch64-linux-gnu
+sudo apt install build-essential
```

Comment:
`build-essential` will install a bit more than what is strictly necessary. `sudo apt install gcc make` would be sufficient.